test/components/field-key/new.spec.js

```diff
@@ -119,7 +120,7 @@ describe('FieldKeyNew', () => {
             .afterResponse(async (app) => {
               await app.get('.field-key-row-popover-link').trigger('click');
               await app.vm.$nextTick();
-              const panel = document.querySelector('.popover .field-key-qr-panel');
+              const panel = document.querySelector('.popover-content-source .field-key-qr-panel');
```

Comment:
Might be within `app` when bootstrap is no longer involved. `app.find` instead of `document.querySelector`